install/amazon_rds/04_configure_the_collector_eks.mdx

```diff
@@ -50,6 +50,39 @@ best practices and use Kubernetes secrets.
 
 <SettingEnvVarsEKS apiKey={props.apiKey} />
 
+When using Kubernetes secrets, it's easier to use the `INI` file style for the
+config. You can find the example of this in the Multiple DB Instances example
+above, where it is used as the value of the `CONFIG_CONTENTS` environment variable.
+
+For instance, you can create a secret called `pganalyze-secrets` and add a key
+`pganalyze-collector.conf`, setting the `INI` collector config as its value.
```

Comment:
Re-reading this part, I think it's the best to have an example yaml file so I'm going to actually add it.